### DISCUSSION TITLE
Why does the font file provided by the website not support Chinese?

### DISCUSSION THREAD

**Discussion by Eve-Middleton** [excalidraw/excalidraw#8931] [General]
I am using [excalidraw](https://excalidraw.com/)，i like it [Excalifont](https://plus.excalidraw.com/excalifont)，I need to download his fonts to install on my PC device. After downloading, I found that the font does not support Chinese, but it can be displayed normally in the  [excalidraw](https://excalidraw.com/), how to  install it?

excalidraw
![PixPin_2024-12-20_09-50-54](https://github.com/user-attachments/assets/0e0e25ed-a78d-4379-9af1-7dc7f0dc98b4)

windows
![PixPin_2024-12-20_10-11-13](https://github.com/user-attachments/assets/dcd7bae9-607b-4239-acbe-cd7f83642656)

**Comment by Mrazator** [comment]
CJK codepoints are technically not part of Excalifont, but we are fallbacking there to Xiaolai font, which can be downloaded here:
https://github.com/lxgw/kose-font/blob/05b1988bcb0bc2ce5787d1d3f292c0d227b93d1b/TTF%20(Simplified%20Chinese)/XiaolaiSC-Regular.ttf

Also, feel free to read more about the CJK support here: https://plus.excalidraw.com/blog/adding-hand-drawn-font-for-chinese-japanese-korean
